tests/src/test/java/alluxio/proxy/s3/S3ClientRestApiTest.java

```diff
@@ -205,6 +209,173 @@ public void deleteNonEmptyBucket() throws Exception {
   }
 
   @Test
+  public void getBucket() throws Exception {
+    final String bucket = "bucket-to-get";
+    AlluxioURI uri = new AlluxioURI(AlluxioURI.SEPARATOR + bucket + AlluxioURI.SEPARATOR);
+    new TestCase(mHostname, mPort, S3_SERVICE_PREFIX + AlluxioURI.SEPARATOR + bucket, NO_PARAMS,
+        HttpMethod.PUT, null, TestCaseOptions.defaults()).run();
+    // Verify the directory is created for the new bucket.
+    Assert.assertTrue(mFileSystemMaster.listStatus(uri, ListStatusOptions.defaults()).isEmpty());
+
+    // Prepare a bucket with direct child objects and objects within sub directories:
```

Comment:
have a common method to set up the testing environment for different kinds of getBucket tests?

Reply:
it's consolidated in another PR